lib/capybara_webkit_builder.rb

```diff
@@ -89,4 +89,17 @@ def build_all
     build &&
     clean
   end
+
+  def make(target = "")
+    env_hide("CDPATH") { sh("#{make_bin} #{target}") }
+  end
+
+  def env_hide(name)
+    @stored_env ||= {}
+    @stored_env[name] = ENV.delete(name)
+
+    yield
+
+    ENV[name] = @stored_env[name]
```

Comment:
I named the method too broadly -- a method named `#env_hide` should restore the environment after the block, but if I just inline it in the `#make` method then I don't need to care. Or something.

I'll do the `ensure`, good idea. Let me know if you think I should inline it or rename it.